packages/web-twig/src/Resources/components/Toast/README.md

```diff
@@ -119,7 +119,7 @@ set the `isCollapsible` prop to `false`.
 👉 Please note that the initial scroll position is always at the **top** of the queue.
 
 ```html
```

Comment:
This is a Prettier error, e.g. it parses this example as HTML (note the ` ```html `). Change also this to `jsx` or `tsx` and it should work correctly.

Reply:
aha! tru!

Reply:
I added another commit where I transfer all code snippets in Twig to use twig syntax

Reply:
👏 Great, thanks 👍